src/views/Statistic/StatisticGroup.js

```diff
@@ -61,7 +60,12 @@ StatisticGroup.propTypes = {
   /** Array of props for Statistic. */
   items: customPropTypes.every([
     customPropTypes.disallow(['children']),
-    PropTypes.array,
+    PropTypes.arrayOf(PropTypes.shape({
+      key: PropTypes.string,
```

Comment:
Turns out the `key` prop is removed by React and is therefore undefined.  When used, these throw browser warnings to use a different prop name to pass the key down.  How about:

```
reactKey
childKey
???
```